mix.exs

```diff
@@ -63,7 +64,7 @@ defmodule Logflare.Mixfile do
       # Phoenix stuff
       {:phoenix, "~> 1.7"},
       # {:phoenix_html, "~> 3.2"},
-      {:phoenix_live_view, "~> 0.18"},
+      {:phoenix_live_view, "~> 0.19"},
```

Comment:
i think this version bump would lead to some errors, especially the `live_component` usage 
https://github.com/phoenixframework/phoenix_live_view/blob/main/CHANGELOG.md#deprecations

Reply:
is the version bump really necessary? might be better to separate this from addition of live dashboard.

Reply:
can revert it and update later. from my testing no regressions have happened in the main flows but nothing against doing it at a separate time